### PR TITLE
Feature/misc spelling corrections

### PR DESCRIPTION
**Pull request by Simran-B**
- FOXX / foxx -> Foxx (if referred to like a brand, but not variables etc.)
- loglevel -> log level
- mountpoint -> mount point
- American spelling, e.g. optimisation -> optimization
- Typos like aquire -> acquire

Mostly touches docs and comments, but also very few places in code:

- https://github.com/arangodb/arangodb/commit/a57cf066989627fd4992e0a58030ee853921993e#diff-b6bc61797479a706dd84c47568d56ab6
- https://github.com/arangodb/arangodb/commit/b262c16de1468b7b22c51f6ffe9749c755194f32#diff-30e4500aa6c680b1f6cdf6322a2d5a48

### CODE REVIEW

File: js/common/modules/@arangodb/foxx/manager-utils.js [hunk 1]
```diff
@@ -219,7 +219,7 @@ function getServiceDefinition (mount) {
 }
 
 // //////////////////////////////////////////////////////////////////////////////
-// / @brief Update the app installed at this mountpoint with the new app
+// / @brief Update the app installed at this mount point with the new app
```
Comment (dothebart) [inline js/common/modules/@arangodb/foxx/manager-utils.js]: do we have mountpoint variables etc. that really should be mountPoint?

Comment (Simran-B) [reply js/common/modules/@arangodb/foxx/manager-utils.js]: It is actually called `mountPoint` **sometimes**:
- `js\apps\system\_admin\aardvark\APP\frontend\js\arango\arango.js`
- some frontend views
- `js\server\modules\@arangodb\actions.js`
- `js\server\modules\@arangodb\foxx\manager.js`

There are quite a few constants that contain `MOUNTPOINT`. Not sure if we want to change that to `MOUNT_POINT` for consistency...

File: arangod/VocBase/LogicalCollection.h [hunk 2]
```diff
@@ -232,6 +232,9 @@ class LogicalCollection: public LogicalDataSource {
   virtual void setStatus(TRI_vocbase_col_status_e);
 
   // SECTION: Serialisation
+  void toVelocyPack(velocypack::Builder&, bool translateCids,
```
Comment (sleto-it) [inline arangod/VocBase/LogicalCollection.h]: what is this one?

**Comment by sleto-it** [comment]
Hi @Simran-B, thanks for for this PR, I have added a comment in a file

Apart from the (easy) conflict to be resolved, this PR touches code, so I think you should start standard PR test on it. Please also get the opinion from @hkernbach as some of the UI code is changed

File: arangod/VocBase/LogicalCollection.h (hunk 2 above)
Comment (Simran-B) [reply arangod/VocBase/LogicalCollection.h]: @sleto-it No clue where this came from. Deleted these lines which don't exist in devel. Should be okay now.

**Pull request merged**